app/scripts/proactive/model/Task.js

```diff
@@ -84,7 +92,8 @@ define(
                     itemType: 'Object',
                     fieldAttrs: {
                         'placeholder': 'genericInformation->info',
-                        "data-help": 'Some extra information about your job often used to change the scheduling behavior for a job. E.g. NODE_ACCESS_TOKEN=rack1 will assign this task to a node with token \"rack1\".'
+                        "data-tab": "Generic Info",
```

Comment:
Task Generic Info 
or Generic Task Info 
To be checked with d.c as well

Reply:
you can show the screenshot in Trello and tag him to have his pov of both remarks